sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/AnalyzeHealthcareTask.java

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.lro;
+
+import com.azure.ai.textanalytics.TextAnalyticsClient;
+import com.azure.ai.textanalytics.TextAnalyticsClientBuilder;
+import com.azure.ai.textanalytics.models.HealthcareEntityCollection;
+import com.azure.ai.textanalytics.models.HealthcareEntityLink;
+import com.azure.ai.textanalytics.models.HealthcareTaskResult;
+import com.azure.ai.textanalytics.models.RecognizeHealthcareEntityOptions;
+import com.azure.ai.textanalytics.models.TextAnalyticsOperationResult;
+import com.azure.ai.textanalytics.models.TextDocumentBatchStatistics;
+import com.azure.ai.textanalytics.models.TextDocumentInput;
+import com.azure.ai.textanalytics.util.RecognizeHealthcareEntitiesResultCollection;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.util.Context;
+import com.azure.core.util.polling.SyncPoller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Sample demonstrates how to analyze a healthcare task.
+ */
+public class AnalyzeHealthcareTask {
+    /**
+     * Main method to invoke this demo about how to begin recognizing the healthcare long-running operation.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        TextAnalyticsClient client =
+            new TextAnalyticsClientBuilder()
+                .credential(new AzureKeyCredential("{key}"))
+                .endpoint("{endpoint}")
+                .buildClient();
+
+        List<TextDocumentInput> documents = new ArrayList<>();
+        for (int i = 0; i < 1; i++) {
```

Comment:
Consider increasing the number as having a for loop for 1 value is weird

Reply:
good catch.